packages/maskbook/src/web3/pipes.ts

```diff
@@ -4,7 +4,7 @@ import { unreachable } from '../utils/utils'
 export function resolveProviderName(providerType: ProviderType) {
     switch (providerType) {
         case ProviderType.Maskbook:
-            return 'Maskbook'
+            return 'Mask'
```

Comment:
Is that stored somewhere? If so, it will be a breaking change

Reply:
Did not stored anywhere.